Zappy_IA_Report/DTC/Code/protocole.py

```
@@ -0,0 +1,210 @@
+    client["is_alive"] = False
+    game.remove_client(client)
+
+
+def handle_fork(client, response) -> None:
+    server_address = client["socket"].getpeername()
+    team_name = client["team_name"]
+    connect_client(server_address, team_name)
+
+
+def handle_Broadcast(client, response) -> None:
+    if (response == "ok"):
```

Pylint (3.3.7)

[convention] Unnecessary parens after 'if' keyword

(C0325)